### PR TITLE
PR #1044 into v5

### PR DESCRIPTION
**Pull request by jaxdesmarais**
### Summary of changes

 - Update changes from #1044 against v5

### Checklist

 - [x] Added a changelog entry
 - [x] Relevant test coverage

### Authors

@jaxdesmarais

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -79,6 +79,7 @@
         * Remove `ThreeDSecureV2BaseCustomization`
         * Remove `CardinalValidateReceiver` from `ThreeDSecureActivity`
         * Make empty `ThreeDSecureLookup` constructor package-private
+        * Add `setCustomFields` to `ThreeDSecureRequest`
```
Comment (sarahkoop) [inline CHANGELOG.md:82]:
```suggestion
```

I think we can just copy over the entire `4.48.0` CHANGELOG (already done in another PR) to include this change since it's not new in v5

**Review by sarahkoop — APPROVED** [review]

**Review by tdchow — APPROVED** [review]
